### PR TITLE
Move Sabberworm\CSS\Parser to vendor_prefixed [MAILPOET-1761]

### PR DESCRIPTION
**Pull request by costasovo**

### CODE REVIEW

File: lib/Config/RequirementsChecker.php [hunk 1]
```diff
@@ -33,8 +33,7 @@ class RequirementsChecker {
     '\pQuery',
     '\Cron\CronExpression',
     '\Html2Text\Html2Text',
-    '\csstidy',
-    '\Sabberworm\CSS\Parser'
+    '\csstidy'
```
Comment (pavel-mailpoet) [inline lib/Config/RequirementsChecker.php]: Rostislav, could you please leave a trailing comma here?

**Comment by pavel-mailpoet** [comment]
@michelleshull this story has changed the class that is used for rendering CSS in forms. So when testing focus on forms stylesheets.

**Comment by michelleshull** [comment]
Approved,  but needs a rebase.

**Comment by costasovo** [comment]
rebased

**Pull request merged**